doc/en/user/source/extensions/monitoring/reference.rst

```diff
@@ -167,12 +167,17 @@ The following attributes are OGC service specific.
    * - Resources
      - Names of resources (layers, processes, etc...) specified as part of the request.
      - List of String
+   * - Resources processing times
+     - Rendering times for resources.
```

Comment:
In years? Minutes? Hours? :-D

Reply:
fixed

Reply:
Forgot to add, this should explain how rendering is done a bit, otherwise people get a wrong impression. Like, the sum of times reported is not the total render time because they overlap, there are two threads, one just reading and preprocessing, the other rendering, while one starts reading a new layer the other is likely still drawing the features of the first. Suggested text: "Rendering is performed by two concurrent threads, one reading and preprocessing data and styles towards a Java2D compatible format, the other painting the results of the first on the canvas. When the first thread starts reading the next layer, the second thread is likely still painting features from the layer before it, thus, times in this list are overlapping with each other, and the sum will be greater than the actual wall rendering time."

Reply:
applied